utils/ccloud_library.sh

```diff
@@ -872,6 +872,12 @@ function ccloud::create_ccloud_stack() {
   REPLICATION_FACTOR=${REPLICATION_FACTOR:-3}
   enable_ksqldb=${1:-false}
 
+  # Check if credit card is on file, which is required for cluster creation
+  if [[ $(ccloud admin payment describe) =~ "Payment method not found" ]]; then
+    echo "ERROR: No credit card on file. Add a payment method and try again."
+    exit 1
+  fi
+
```

Comment:
Would only searching for "not found" be a little more tolerant to potential changes in this simple text output conditional check?

```suggestion
  # Check if credit card is on file, which is required for cluster creation
  if [[ $(ccloud admin payment describe) =~ "not found" ]]; then
    echo "ERROR: No credit card on file. Add a payment method and try again."
    exit 1
  fi

```